### PR TITLE
Blue button test and less text on button tests

### PR DESCRIPTION
**Pull request by beccaklam**
Created tests for donate.mozilla.org to see if blue donate buttons and less text on the donate buttons will improve conversion on the page.

### CODE REVIEW

File: less/components/payment-buttons.less [hunk 1]
```diff
@@ -21,6 +21,19 @@
   }
 }
 
+.credit-card-logos {
```
Comment (cadecairos) [inline less/components/payment-buttons.less]: I'm curious why this was moved? none og the rules look different

**Comment by cadecairos** [comment]
@beccaklam how do I trigger the tests?

**Comment by cadecairos** [comment]
aha, nevermind, found the right string: `http://localhost:3000/en-US/?test=buttons-less-text`

**Comment by beccaklam** [comment]
@cadecairos Yeah the tests are ?test=buttons-less-text and ?test=donate-buttons-blue

**Comment by beccaklam** [comment]
@cadecairos Checking now.

**Comment by beccaklam** [comment]
@cadecairos Ok, I fixed the first bug you flagged in the other pull request: "Fixed no amex on blue buttons". Wasn't sure how to merge that request into this one? If someone can show me how I can do that next time :)

**Pull request merged**